.github/workflows/ai-review.yml

```
@@ -0,0 +1,28 @@
+
+jobs:
+  review:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write # 必须开启写入权限才能发评论
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v4
+
+      - name: Gemini AI Review
+        uses: google-gemini-github-action@v1 # 或者是 marketplace 搜到的 gemini-ai-code-review-action
```

actionlint (1.7.12)

[error] specifying action "google-gemini-github-action@v1" in invalid format because owner is missing. available formats are "{owner}/{repo}@{ref}" or "{owner}/{repo}/{path}@{ref}"

(action)